infracost-usage-defaults.large.yml

```diff
@@ -777,6 +777,14 @@ resource_type_default_usage:
     monthly_data_retrieval_gb: 2000 # Monthly number of data retrieval in GB.
     snapshots_storage_gb: 880 # Total size of Snapshots in GB
     metadata_at_rest_storage_gb: 670 # Total size of Metadata in GB
+  azurerm_storage_table:
```

Comment:
I've added these example files. We aim for the following:

In `infracost-usage-defaults.large.yml` each cost component should equal around $20 worth.
In `infracost-usage-defaults.medium.yml` each cost component should equal around $10 worth
In `infracost-usage-defaults.small.yml` each cost component should equal around $5 worth